src/pages/Home.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { Container } from '@material-ui/core';
+import ScholarshipsMadeSimpleSection from '../components/ScholarshipsMadeSimpleSection';
 
 function Home() {
   return (
-    <Container style={{ textAlign: 'center' }}>
-      <h1>Find Scholarships Today</h1>
+    <Container style={{ textAlign: 'left' }}>
```

Comment:
You shouldn't have to textAlign left. That should be the default. Inspect the document in the browser to figure out what's causing the non-left alignment.

Reply:
ok